docs/contributing.rst

```diff
@@ -67,14 +67,17 @@ The following tools are there to help you:
 Integration
 ~~~~~~~~~~~
 
-First, install `Go`_ 1.5 and start Boulder_, an ACME CA server::
+First, install `Go`_ 1.5 (pick a value for GOPATH and put $GOPATH/bin in your
+PATH), libtool-ltdl, mariadb-server and rabbitmq-server and then start
+Boulder_, an ACME CA server::
 
   ./tests/boulder-start.sh
 
-The script will download, compile and run the executable; please be
-patient - it will take some time... Once its ready, you will see
-``Server running, listening on 127.0.0.1:4000...``. You may now run
-(in a separate terminal)::
+The script will download, compile and run the executable; please be patient -
+it will take some time... Once its ready, you will see ``Server running,
+listening on 127.0.0.1:4000...``. Add the ``venv/bin/`` subdirectory of your
```

Comment:
You don't need to add `venv/bin` to `PATH` - this is automatically done by `source ./venv/bin/activate` step